src/@types/partials.ts

```diff
@@ -91,3 +91,11 @@ export interface IUsageLimit {
     // eslint-disable-next-line camelcase
     admin_contact?: string;
 }
+
+export enum Membership {
```

Comment:
I'm not sure an enum is neccessarily best for this sort of data type: enums get annoyingly large when transpiled and we're using it as a simple mapping: a plain `type` might be more suitable here instead.

Reply:
I'm not sure that it is going to be that large for just 5 items, but from my point of view enums are much easier and cleaner from the readability and code maintenance perspective

Reply:
Ther ewas a time when I'd agree, though enums appear to be the way we've gone regardless. At 5 items, it's about 25 lines of boilerplate - enums make a lot more sense when it's >10-15 items.

Reply:
const enums are better at this